src/main/java/org/calypsonet/terminal/calypso/card/CalypsoCardSelection.java

```diff
@@ -34,7 +33,7 @@
  * checked for the various parameters:
  *
  * <ul>
- *   <li>SFI: [0..31] (0 indicates the current EF)
+ *   <li>SFI: [0..30] (0 indicates the current EF)
  *   <li>Record number: [1..255]
```

Comment:
- record number must be in range [1..250]
 - counter number must be in range [1..83],   because 250 / 3 => 83
 - Offset: [0..249] or [0..32767] for binary files
 - Input data length: [1..250] or [1..32767] for binary files